monitor/service/api/__init__.py

```diff
@@ -44,10 +44,18 @@
         mesos_username = config.get('spark_mesos', 'mesos_username')
 
     if 'external_api' in plugins:
+
+        # Setting default value
+        k8s_manifest = "./data/conf"
```

Comment:
define in a constant to make it easier for future contributors/users.

Reply:
Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d